addons/hr_recruitment/wizard/applicant_send_mail_views.xml

```diff
@@ -15,9 +15,16 @@
                 <field name="body" nolabel="1" class="oe-bordered-editor"
                         placeholder="Write your message here..."
                         options="{'style-inline': true}" force_save="1"/>
+                <group>
+                            <field name="attachment_ids" widget="many2many_binary" string="Attach a file" nolabel="1" colspan="2"/>
+                            <field name="template_id" string="Load template" options="{'no_create': True}"/>
+                </group>
                 <footer>
                     <button name="action_send" string="Send" type="object" class="btn-primary" data-hotkey="q"/>
                     <button string="Cancel" class="btn-secondary" special="cancel" data-hotkey="x"/>
+                    <button icon="fa-lg fa-save" type="object"
+                                name="action_save_as_template" string="Save as new template"
+                                class="float-end btn-secondary" help="Save as a new template" data-hotkey="w"/>
```

Comment:
nitpicking but misaligned :innocent: 
Also, the help doesn't seem needed as it's exatly the same as the string
```suggestion
                    <button icon="fa-lg fa-save" type="object"
                        name="action_save_as_template" string="Save as new template"
                        class="float-end btn-secondary" data-hotkey="w"/>
```